500x series

znicholls opened the pull request:
A clean set of notebooks that clarifies how things work at the expense of processing any extra datasets. This may end up being a 'reset', but for now it's just a clean up that doesn't touch existing notebooks.

eahoegner commented:
minor comment: 
a path in `constants_5000.py ` needs to be updated to:
`ADAM_ET_AL_2024_RAW_PATH = DATA_ROOT / "global" / "adam-et-al-2024" / "data_raw"`

znichollscr commented:
Oops my bad I didn't add the folder where I moved the data. Done now in the latest commit

jkikstra commented:
Notes (list will be appended as I go):
* file missing: `nmvoc_aux_conversion = pd.read_excel(GFED4_RAW_PATH / "NMVOC-species.xlsx", index_col=0)` -> `No such file or directory: '{path}\\emissions_harmonization_historical\\data\\raw\\gfed4\\NMVOC-species.xlsx'`
*

eahoegner commented:
and for script `5096` that runs MAGICC we need to change the linux in `        elif platform.system().startswith("linux"): ` to Linux (lines 162 and 181)

znicholls commented:
> files missing for GFED

Both added

> Memory error when aggregating to countries

Hmm tricky, let's see if we care or not.

> Current notes to self

Added all of this to the README

> wonder if it's worth including 5030 into the python script by default?

Updated and reordered

> platform.system().startswith("linux")

updated

Merging now, cross your fingers!
